py_mmd_tools/mmd_elements.yaml

```diff
@@ -150,19 +150,23 @@ use_constraint:
     acdd: license
   resource:
     maxOccurs: '1'
-    acdd: license_resource
+    acdd_ext: license_resource
 
 personnel:
   maxOccurs: unbounded
   minOccurs: '0'
   role:
     default: 'unknown'
     separator: ','
-    acdd: creator_role
+    acdd: 
+      - creator_role
+      - contributor_role
   name:
     default: 'unknown'
     separator: ','
-    acdd: creator_name
+    acdd:
+      - creator_name
+      - contributor_name
```

Comment:
I think we need contributor_email and contributor_organisation also else I get like this:
```
  <mmd:personnel>
    <mmd:role>Metadata author</mmd:role>
    <mmd:name>TESTTESTTEST</mmd:name>
    <mmd:email>unknown</mmd:email>
    <mmd:organisation>unknown</mmd:organisation>
  </mmd:personnel>
```

Reply:
added now